app/templates/service_update_audits.html

```diff
@@ -0,0 +1,154 @@
+                <tr>
+                    <th>Supplier</th>
+                    <th>Date</th>
+                    <th>Diff</th>
+                    <th></th>
+                </tr>
+                </thead>
+                <tbody class="summary-item-body">
+                {% for audit_event in audit_events %}
+                <tr class="summary-item-row">
+                    <td class="summary-item-field-name">
+                        {{ audit_event.data.supplierName }}
+                    </td>
+                    <td class="summary-item-field-name">
+                        {{ audit_event.createdAt|timeformat }}
+                        <br/>
+                        {{ audit_event.createdAt|dateformat }}
+                    </td>
+                    <td class="summary-item-field-name">
+                        <a href="{{ url_for('.service_update_audits') }}">View changes</a>
+                    </td>
+                    <td class="summary-item-field-name">
+                        {% if not audit_event.acknowledged %}
+                        <form action="{{ url_for('.submit_service_update_acknowledgment', audit_id=audit_event.id) }}" method="post">
+                            <div style="display:none;">
+                                <input name="csrf_token" type="hidden" value="{{ csrf }}">
+                                <input name="acknowledged" type="hidden" value="{{ acknowledged }}">
+                                <input name="audit_date" type="hidden" value="{% if form.audit_date.data %}{{ form.audit_date.data }}{% endif %}">
+                            </div>
+                            <button class="button-secondary">Acknowledge</button>
+                        </form>
+                        {% else %}
+                            {{ audit_event.acknowledgedBy }}
+                            <br/>
+                            {{ audit_event.createdAt|timeformat }}
+                            <br/>
+                            {{ audit_event.createdAt|dateformat }}
+                            {% endif %}
+                    </td>
+                </tr>
+                {% endfor %}
+                </tbody>
+            </table>
+            <!--
+            <nav role="navigation">
+                <ul class="previous-next-navigation">
+
+                    <li class="previous">
+                        <a href="">
+                            Previous <span class="visuallyhidden">page</span>
+                            <span class="page-numbers">1 of 3</span>
+                        </a>
+                    </li>
+
+                    <li class="next">
+                        <a href="">
+                            Next <span class="visuallyhidden">page</span>
+                            <span class="page-numbers">2 of 3</span>
+                        </a>
+                    </li>
+                </ul>
+            </nav>
+            -->
```

Comment:
My little brain doesn't understand. I'm assuming this is necessary because it's the only place the pagination navigation is used but it's commented out. How does it work?